components/button/button-add.js

```diff
@@ -54,8 +54,9 @@ class ButtonAdd extends RtlMixin(PropertyRequiredMixin(FocusMixin(LocalizeCoreEl
 				cursor: pointer;
 				display: flex;
 				font-family: inherit;
-				height: 0.9rem;
+				height: 11px;
```

Comment:
setting height to be 1px (line) + 5 px on each side of the line.
The margins below are to prevent nearby content from overlapping with the + / + and text